apps/weshipit/pages/react-native-tools/index.tsx

```diff
@@ -35,6 +35,44 @@ export default function ReactNativeApiPage({ records }) {
       </div>
 
       <div className="mx-auto max-w-screen-2xl px-4 pb-48 sm:px-6">
+        <form className="py-8">
+          <label className="sr-only mb-2 text-sm font-medium text-gray-900 dark:text-white">
+            Search
+          </label>
+          <div className="relative">
+            <div className="pointer-events-none absolute inset-y-0 left-0 flex items-center pl-3">
+              <svg
+                aria-hidden="true"
+                className="h-5 w-5 text-gray-500 dark:text-gray-400"
+                fill="none"
+                stroke="currentColor"
+                viewBox="0 0 24 24"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  stroke-width="2"
+                  d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
+                ></path>
+              </svg>
+            </div>
+            <input
+              type="search"
+              id="default-search"
+              className="block w-full rounded-lg border border-gray-300 bg-gray-50 p-4 pl-10 text-sm text-gray-900 focus:border-blue-500 focus:ring-blue-500 dark:border-gray-600 dark:bg-gray-700 dark:text-white dark:placeholder:text-gray-400 dark:focus:border-blue-500 dark:focus:ring-blue-500"
+              placeholder="Search..."
+              required
```

Comment:
drop the `form` tag and add a `onChange={handleChange}` feature
like this example https://dev.to/asimdahall/simple-search-form-in-react-using-hooks-42pg